### PR TITLE
[EspaceCandidate] Add managed users list

### PR DESCRIPTION
**Pull request by illusionOfParadise**
This PR related to https://github.com/EnMarche/worker/pull/39

### CODE REVIEW

**Comment by illusionOfParadise** [comment]
[ch12623]

File: src/Entity/Projection/ManagedUser.php [hunk 1]
```diff
@@ -70,6 +70,13 @@ class ManagedUser
      */
     private $email;
 
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(length=150, nullable=true)
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply src/Entity/Projection/ManagedUser.php:77]: It's the same as for the address field of the adherent entity.

File: src/Exporter/ManagedUsersExporter.php [hunk 2]
```diff
@@ -19,32 +19,46 @@ public function __construct(SonataExporter $exporter, ManagedUserRepository $rep
...
-            'Genre' => 'getGenderLabel',
-            'Rôle' => 'getUserRoleLabels',
-            'Commune' => 'city',
-            'Code postal' => 'postalCode',
-            'Pays' => 'country',
-            'Adhésion le' => 'createdAt',
-            'Comités' => 'getCommitteesAsString',
-        ];
+        switch ($spaceType) {
+            case 'candidate':
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply src/Exporter/ManagedUsersExporter.php]: Done

File: src/Form/ManagedUsers/CandidateManagedUsersFilterType.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    {
+        $builder
+            ->add('gender', GenderType::class, [
+                'placeholder' => 'Tous',
+                'expanded' => true,
+                'required' => false,
+            ])
+            ->add('ageMin', IntegerType::class, ['required' => false])
+            ->add('ageMax', IntegerType::class, ['required' => false])
+            ->add('firstName', TextType::class, ['required' => false])
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply src/Form/ManagedUsers/CandidateManagedUsersFilterType.php]: Done

File: src/Repository/Geo/ZoneRepository.php [hunk 4]
```diff
@@ -62,4 +62,24 @@ public function createSelectForCandidatesQueryBuilder(): QueryBuilder
             ])
         ;
     }
+
+    public function createSelectByManagedZones(array $zones): QueryBuilder
```
Comment (ottaviano) [inline src/Repository/Geo/ZoneRepository.php:66]: seems to be unused

File: templates/candidate/_layout.html.twig [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    {% if last_visit_date is not null %}
+        • Dernière visite le <span class="text--black text--bold">{{ last_visit_date|date('d/m/Y') }}</span>
+        à <span class="text--black text--bold">{{ last_visit_date|date('H\\hi') }}</span>
+    {% endif %}
+{% endblock %}
+
+{% block final_javascripts %}
+    {{ parent() }}
+
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
```
Comment (ottaviano) [inline templates/candidate/_layout.html.twig:47]: do we really need to include jquery here?

File: templates/candidate/_sidebar.html.twig [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{% if isCandidate or is_granted('HAS_DELEGATED_ACCESS_ANY') %}
```
Comment (ottaviano) [inline templates/candidate/_sidebar.html.twig:1]: why we check isCandidate here? this template is used on `candidate/layout`, so we know that the user is already candidate, no?

File: templates/candidate/_table.html.twig [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+            <td>
+                {% set hasSmsSubscriptionType = user.hasSmsSubscriptionType %}
+                <span class="status status__{{ hasSmsSubscriptionType ? '1' : '2' }}">
+                    {{ (hasSmsSubscriptionType ? 'common.adherent.subscribed' : 'common.adherent.unsubscribed')|trans }}
+                </span>
+            </td>
+        </tr>
+    {% else %}
+        <tr>
+            <td colspan="8" class="text--center">
```
Comment (ottaviano) [inline templates/candidate/_table.html.twig:61]: `colspan="5"` :)

Comment (illusionOfParadise) [reply templates/candidate/_table.html.twig:61]: it's my favorite 😂

File: templates/candidate/_sidebar.html.twig (hunk 6 above)
Comment (illusionOfParadise) [reply templates/candidate/_sidebar.html.twig]: Yes, it has been removed.

File: templates/candidate/_layout.html.twig (hunk 5 above)
Comment (illusionOfParadise) [reply templates/candidate/_layout.html.twig:47]: Yes, for working list template used by all other spaces.

**Review by ottaviano — APPROVED** [review]

**Pull request merged**